components/feature/awesomebar/src/main/java/mozilla/components/feature/awesomebar/provider/SessionSuggestionProvider.kt

```diff
@@ -48,4 +49,17 @@ class SessionSuggestionProvider(
 
         return suggestions
     }
+
+    private fun Session.contains(text: String) =
+        (url.contains(text, ignoreCase = true) || title.contains(text, ignoreCase = true))
+
+    private fun shouldIncludeSelectedSession(session: Session): Boolean {
+        return if (excludeSelectedSession) {
+            !(isSelectedSession(session) ?: false)
```

Comment:
nit: this is pretty hard to read. `isSelectedSession` probably should just return a non-null boolean.

Reply:
Updated 👍